backend/editor/entries.py

```diff
@@ -101,4 +186,40 @@ def update_nodes(label, entry, incomingData):
 
     params = dict(incomingData, id=entry)
     result = session.run(" ".join(query), params)
-    return result
\ No newline at end of file
+    return result
+
+def update_node_children(entry, incomingData):
+    """
+    Helper function used for updation of node children with given id 
+    """
+    # Parse node ids from Neo4j Record object
+    current_children = [record["b.id"] for record in list(get_children(entry))]
+    deleted_children = list(set(current_children) ^ set(incomingData))
+
+    # Delete relationships
+    for child in deleted_children:
+        query = []
+        main_language_code = child[:2]

```

Comment:
Here again, use split instead of lenght of language code to stick to the spec.
```suggestion
        main_language_code = child.split(",", 1)[0]
```